### PR TITLE
Fix/26 update Readme

### PR DESCRIPTION
**Pull request by beatrizsanchezt**
Fix issue https://github.com/12358-lab/my-portfolio/issues/26

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,68 +1,34 @@
-This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+# Welcome to my Portfolio
 
-## Available Scripts
+This project was made using React and Elastic UI as a learning practice.
```
Comment (nestor-diaz) [inline README.md]: Make React and Elastic UI links to the projects

File: README.md [hunk 2]
```diff
@@ -1,68 +1,34 @@
...
 
-In the project directory, you can run:
+This is the latest version is deployed here: www.beatrizsanchez.dev
```
Comment (nestor-diaz) [inline README.md]: `The latest version...`

Comment (nestor-diaz) [reply README.md]: Change the address to `http://beatrizsanchez.dev` and make it a link

File: README.md [hunk 3]
```diff
@@ -1,68 +1,34 @@
...
-### `npm start`
+![My Portfolio image](src/assets/images/myPortfolioEducation.png)
 
-Runs the app in the development mode.<br />
-Open [http://localhost:3000](http://localhost:3000) to view it in the browser.
+## Setup
 
-The page will reload if you make edits.<br />
-You will also see any lint errors in the console.
+To run this project, install it locally using npm:
```
Comment (nestor-diaz) [inline README.md]: *To run this project locally using npm:

File: README.md [hunk 4]
```diff
@@ -1,68 +1,34 @@
...
 
-### `npm test`
+```
+$ cd ../myPortfolio
```
Comment (nestor-diaz) [inline README.md]: This is not needed... whoever clones you repo might pick a different directory.

File: README.md [hunk 5]
```diff
@@ -1,68 +1,34 @@
...
+$ npm install
+$ npm start
+```
 
-Launches the test runner in the interactive watch mode.<br />
-See the section about [running tests](https://facebook.github.io/create-react-app/docs/running-tests) for more information.
+## Features
 
-### `npm run build`
+- **About**: A short description about myself
```
Comment (nestor-diaz) [inline README.md]: Dot at the end of the line

File: README.md [hunk 6]
```diff
@@ -1,68 +1,34 @@
...
+- **Education**: Cards was used to display diplomas and certifications.
```
Comment (nestor-diaz) [inline README.md]: `Cards were`

File: README.md [hunk 7]
```diff
@@ -1,68 +1,34 @@
...
+- **Skills**: A table with sort option by column header, shows my skilss and levels. Hooks were implemented here.
```
Comment (nestor-diaz) [inline README.md]: `A table with sorting by column.... Hooks were used here`.

File: README.md [hunk 8]
```diff
@@ -1,68 +1,34 @@
...
-
-### Deployment
-
-This section has moved here: https://facebook.github.io/create-react-app/docs/deployment
-
-### `npm run build` fails to minify
-
-This section has moved here: https://facebook.github.io/create-react-app/docs/troubleshooting#npm-run-build-fails-to-minify
+- Unit Testing with Jest.
+- I want to change Project section and read my projects from Github.
```
Comment (nestor-diaz) [inline README.md]: `the Project... and fetch my projects from GitHub`

**Review by nestor-diaz — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (beatrizsanchezt) [reply README.md]: Updated

File: README.md (hunk 2 above)
Comment (beatrizsanchezt) [reply README.md]: Updated

File: README.md (hunk 3 above)
Comment (beatrizsanchezt) [reply README.md]: Updated

File: README.md (hunk 5 above)
Comment (beatrizsanchezt) [reply README.md]: Updated

File: README.md (hunk 6 above)
Comment (beatrizsanchezt) [reply README.md]: Updated

File: README.md (hunk 7 above)
Comment (beatrizsanchezt) [reply README.md]: Updated

File: README.md (hunk 8 above)
Comment (beatrizsanchezt) [reply README.md]: Updated

File: README.md (hunk 4 above)
Comment (beatrizsanchezt) [reply README.md]: Removed

File: README.md [hunk 9]
```diff
@@ -1,26 +1,25 @@
 # Welcome to my Portfolio
 
-This project was made using React and Elastic UI as a learning practice.
+This project was made using [React](https://reactjs.org/) and [Elastic UI](https://elastic.github.io/) as a learning practice.
 
-This is the latest version is deployed here: www.beatrizsanchez.dev
+The latest version is deployed [here](https://www.beatrizsanchez.dev).
```
Comment (nestor-diaz) [inline README.md]: It's unusual to include `www` in the addresses

**Review by nestor-diaz — CHANGES_REQUESTED** [review]

File: README.md (hunk 9 above)
Comment (beatrizsanchezt) [reply README.md]: Updated

**Review by nestor-diaz — APPROVED** [review]

**Pull request merged**